dspace/config/modules/rest.cfg

```diff
@@ -4,6 +4,18 @@
 # These configs are used by the REST module                     #
 #---------------------------------------------------------------#
 
+# Sets the schemes for the API URLs (http, https).
+rest.schemes=${rest.schemes}
+
+# Sets the host (including a port) for the API URLs. Does not include the schemes nor context root.
+rest.host=${rest.host}
+
+# Sets the context root for the API calls.
+rest.setBasePath=${rest.setBasePath}
```

Comment:
These settings cause DSpace to have a circular default reference (a bad practice). This is the *old* way (build.properties), and is no longer supported or recommended.

Instead, the default values should be placed in this `rest.cfg` file. They can always be overridden by copying the config into your local.cfg.